lib/voorhees/json_api.ex

```diff
@@ -50,4 +77,21 @@ defmodule Voorhees.JSONApi do
 
     actual
   end
+
+  defp _stringify_items([]), do: []
```

Comment:
I already have something similar to this elsewhere, we should combine

https://github.com/danmcclain/voorhees/blob/master/lib/voorhees.ex#L145-L155